ui/app/selectors/transactions.js

```
@@ -290,21 +307,21 @@ export const nonceSortedTransactionsSelector = createSelector(
             nonceProps.hasCancelled = true;
           }
         } else {
-          nonceToTransactionsMap[nonce] = {
-            nonce,
+          nonceToTransactionsMap[nonceKey] = {
```

ESLint

[error] Delete `··`

(prettier/prettier)